### PR TITLE
[W04H03] Added more quickhull tests

### PR DESCRIPTION
**Pull request by HendrikHuebner**

### CODE REVIEW

**Review by nilsreichardt — COMMENTED** [review]
Hey @HendrikHuebner 👋 I have not reviewed the content of you PR but the one check is failing because you are using round brackets instead of square brackets. So changing "(w04h03)" to "[W04H03]" would pass the check :+1:

**Review by ipadmini22 — APPROVED** [review]
I'd say the tests are reasonable and add some certainty for diagonal and squares. So why not :)

**Comment by HendrikHuebner** [comment]
> Hey @HendrikHuebner 👋 I have not reviewed the content of you PR but the one check is failing because you are using round brackets instead of square brackets. So changing "(w04h03)" to "[W04H03]" would pass the check 👍

Fixed it.

File: w04h03/test/pgdp/AdditionalQuickHullTests.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package pgdp.convexhull;
```
Comment (nilsreichardt) [inline w04h03/test/pgdp/AdditionalQuickHullTests.java:1]: Are you sure that this is the correct import (under the condition that this file is located under `w04h03/test/pgdp/AdditionalQuickHullTests.java`)?

I needed to change the package to the following line to get it to work.

```suggestion
package pgdp;
```

**Review by nilsreichardt — APPROVED** [review]
LGTM

Just a note: I personally try to avoid using `for` loops in tests because it makes them very hard to debug if the test failed.

**Pull request merged**